src/JuMP.jl

```diff
@@ -466,15 +466,30 @@ function Base.setindex!(m::JuMP.Model, value, name::Symbol)
     m.objdict[name] = value
 end
 
-# usage warnings
+"""
+    operator_warn_with_variable(vref::AbstractVariableRef)
```

Comment:
Why not call this on one of the expressions directly instead of one of the variables? It seems unusual to pick out the variable just to dispatch to the right model.